stable_baselines3/common/vec_env/vec_video_recorder.py

```diff
@@ -93,7 +92,7 @@ def step_wait(self) -> VecEnvStepReturn:
             self.video_recorder.capture_frame()
             self.recorded_frames += 1
             if self.recorded_frames > self.video_length:
-                logger.info("Saving video to ", self.video_recorder.path)
+                print(f"Saving video to {self.video_recorder.path}")
```

Comment:
Hm, `logger` did seem more natural here, I guess we're doing this to try and avoid using the global logger? Perhaps we can make the `common.logger` module only used for key-value logging (like recording loss over time), but still have some global logger for the Stable Baselines code. Or even just logger = logging.getLogger(__name__) in each module.

Reply:
I'm a bit puzzled with this one as it is the only place where logger is called from outside an algorithm.

>Or even just logger = logging.getLogger(name) in each module.

this seems a bit overkill compared to current usage, no?

Reply:
Yeah it is a bit overkill for a single usage. Nice since it allows users of the library to e.g. choose what log level they want for Stable Baselines as a whole. Given this is a one-off I think I'm fine with sticking to print, or just `import logging` and `logging.info`